### ISSUE TITLE
Blender crashes system on debian 11 when attempting to move around camera

### ISSUE THREAD

**Issue by ServerBBQ** [flathub/org.blender.Blender#136]
Specs:
```
OS: Debian GNU/Linux 11 (bullseye) x86_64 
Kernel: 5.10.0-21-amd64 
Resolution: 1920x1080 
DE: GNOME 3.38.6 
CPU: AMD Ryzen 5 5600 (12) @ 3.500GHz 
GPU: AMD ATI Radeon RX 5600 XT
Memory: 32014MiB
```

Troubleshooting steps:
* installed blender from official website and done steps below
   * worked so this narrows it down to the flatpak being the * problem
* disabled gpu acceleration via flatseal
   * worked but very laggy
* booted from live iso file to see if the issue is related to any of the packages installed on my system
   *  showed same issue as on my normal install

Steps to reproduce:
1. boot off of the debian-live-11.6.0-amd64-gnome+nonfree iso(https://cdimage.debian.org/cdimage/unofficial/non-free/cd-including-firmware/11.6.0-live+nonfree/amd64/bt-hybrid/)
2. install flatpak using instructions from https://flatpak.org/setup/Debian and logout instead of restarting to complete setup
3. install flatseal and blender from flathub
4. go into flatseal and disable wayland for blender(blender doesn't launch on wayland with gnome 3.38)
5. launch blender, open a demo file(I used Lone Monk from https://www.blender.org/download/demo-files/)
6. move camera around and observe crash 

Image of crash:
![image](https://user-images.githubusercontent.com/76498501/233525238-6318c43d-b8a3-4c1b-9521-946ff3739401.png)

**Comment by kevinsmia1939** [comment]
By crashes system, did you mean the whole OS crash or just blender?
Can you run blender in terminal to see if there is any error?
`flatpak run org.blender.Blender `

**Comment by kevinsmia1939** [comment]
It is rather hard to tell, it could be issue with blender that show memory leak.

**Comment by poperigby** [comment]
I'm having the same issue on Fedora Kinoite 38 with an RX 5700 XT.

**Comment by Rex-odus** [comment]
Yeah. Me too. I crashes in the background while doing other things. It doesn't respond to close in KDE. Only kill can remove it. And than of course all your work is gone. And as icing on the cake, there is no log that says anything useful.

If no fix appears soon, I'll have to go back to Windows. Because only Debian has LUKS in the installer these days (for custom partition management). The whole Ubuntu and downwards fork is broken (what a stupid morons to bring it to the market anyway).

Debian 12
NVidia 535.183.01
Blender 4.2 LTS
64GB RAM
